init/relabel-rw.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash --
```

Comment:
What bash features you use here? I'd prefer /bin/sh (and consequently shellcheck noticing non-POSIX features) in scritpts that don't absolutely need bash (for example to have proper arrays).